### PR TITLE
Homes Task 1

### PR DESCRIPTION
**Pull request by suyash929**
@sriram15690 , please review the home task of HTML/CSS

### CODE REVIEW

File: Assignment1/homeTask.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                <span class="newsTitle"><b>Title One</b></span>
+                <br>
+                <span class="newsDate">Posted on <b>29 June,2019</b> // Category: <b>Category One</b></span>
+                <p class="newsDetail">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+                <input type="button" class="readingButton" value="Continue Reading">
+            </div>
+        </div>
+        <hr class="subDivision">
+        <div class="descriptionSection">
+            <img class="newsImg" src="image.jpg"/>
```
Comment (sriram15690) [inline Assignment1/homeTask.html]: image must have "alt" attribute

File: Assignment1/homeTask.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            <div class="newsDesc">
+                <span class="newsTitle"><b>Title One</b></span>
+                <br>
```
Comment (sriram15690) [inline Assignment1/homeTask.html:71]:
```
<br> or <br/>.
```
 pls check

File: Assignment1/main.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        width:43%;
+    }
+    .three{
+        width:36%;
+    }
+    .emailBox{
+        width: 88%;
+    }
+    .subscribeButton{
+        font-size: 11px;
```
Comment (sriram15690) [inline Assignment1/main.css]: check about px vs em vs rem

File: Assignment1/main.css [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+.newsTitle{
+    color:  #273852;
+    padding-bottom: 1%;
+    font-size: 1.8rem;
+}
+.newsDate{
+    font-size:0.7rem;
+}
+.newsDetail{
+    font-size:0.9rem ;
```
Comment (sriram15690) [inline Assignment1/main.css:89]: Here there rem & at the bottom I see px. Pls check

**Review by sriram15690 — CHANGES_REQUESTED** [review]

File: Assignment1/homeTask.html (hunk 2 above)
Comment (suyash929) [reply Assignment1/homeTask.html:71]: Checked this, actually in HTML5 we don't need closing slash, before we needed it like in XHTML1.0

File: Assignment1/main.css (hunk 4 above)
Comment (suyash929) [reply Assignment1/main.css:89]: ok

File: Assignment1/main.css (hunk 3 above)
Comment (suyash929) [reply Assignment1/main.css]: ok

File: Assignment1/homeTask.html (hunk 1 above)
Comment (suyash929) [reply Assignment1/homeTask.html]: I will add this attribute

**Pull request merged**